include/onnxruntime/core/framework/ml_value.h

```diff
@@ -83,6 +85,10 @@ struct OrtValue {
     fence_ = v.fence_;
   }
 
+  bool HasElement() const noexcept {
+    return data_ != nullptr;
```

Comment:
nit: 'element' is confusing given a tensor has 'elements' and an 'element type'. HasValue would be more consistent with std::optional but HasData would also be fine.

Reply:
Made it HasValue()